applications/nrf_desktop/nrf_desktop_integration.rst

```diff
@@ -0,0 +1,953 @@
+.. _nrf_desktop_porting_guide:
+
+Integrating your own hardware
+#############################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This section describes how to adapt the nRF Desktop application to different hardware, and lists steps required for adding a new board.
+
+.. _porting_guide_adding_board:
+
+Adding a new board
+******************
+
+When adding a new board for the first time, focus on a single configuration.
+Moreover, keep the default debug build type that the application is built with, and do not add any additional build type parameters.
+The following procedure uses the gaming mouse configuration as an example.
+
+Zephyr support for a board
+==========================
+
+Before introducing nRF Desktop application configuration for a given board, you need to ensure that the board is supported in Zephyr.
+
+.. note::
+   You can skip this step if your selected board is already supported in Zephyr.
+
+Follow the Zephyr's :ref:`zephyr:board_porting_guide` for detailed instructions related to introducing Zephyr support for a new board.
+Make sure that the following conditions are met:
+
+#. Edit the DTS files to make sure they match the hardware configuration.
+   Pay attention to the following elements:
+
+   * Pins that are used.
+   * Bus configuration for optical sensor.
+   * `Changing interrupt priority`_.
+
+#. Edit the board's Kconfig files to make sure they match the required system configuration.
+   For example, disable the drivers that will not be used by your device.
+
+.. tip::
+   You can define the new board by copying the nRF Desktop reference design files that are the closest match for your hardware and then aligning the configuration to your hardware.
+   For example, for gaming mouse use :file:`nrf/boards/arm/nrf52840gmouse_nrf52840`.
+
+nRF Desktop configuration
```

Comment:
Question - would it make sense to change it to "Configuring nRF Desktop"/"Configure nRF Desktop" to highlight that these are configuration steps rather than a list of configurations (also to differentiate it from other sections with the same name)?

Reply:
Consider `nRF Desktop support for a board` (section is next to `Zephyr board support` and describes how to add nRF Desktop specific configuration files for a given board).

Reply:
How about unifying this naming for both sections, while still implying through the naming that there are steps to be performed? E.g., 
1. Adding Zephyr board support 
2. Adding nRF Desktop board support ?

Reply:
After offline discussion, updated as : nRF Desktop support for a board.